core-web/libs/block-editor/src/lib/shared/utils/suggestion.utils.ts

```diff
@@ -26,6 +26,14 @@ const headings = [...Array(3).keys()].map((level) => {
     };
 });
 
+const image = [
```

Comment:
could you type the object in the future? {label: string, icon: string, id: string}

Reply:
Let's do it now. There is a type for this called `DotMenuItem`